src/transformers/file_utils.py

```diff
@@ -496,3 +498,253 @@ def _resumable_file_manager():
             json.dump(meta, meta_file)
 
     return cache_path
+
```

Comment:
should we create a new `benchmarking_utils.py` file for this?

Reply:
indeed!